src/pages/SignUp/SignUp.scss

```
@@ -0,0 +1,110 @@
+input {
+  padding: 22px 20px;
+}
+
+.signUp {
+  width: 400px;
+  margin: 0 auto;
+  padding-top: 80px;
+  .signUpTitle {
+    border-bottom: 1px solid lightgray;
+    font-size: 32px;
+    text-align: center;
+    font-weight: 700;
+    line-height: 42px;
+    padding-bottom: 40px;
+  }
+  .infTitle {
+    font-size: 20px;
+    font-weight: 700;
+    padding: 30px 0;
+  }
+  .signUpInputContainer {
+    list-style: none;
+    padding: 0;
+    .errorMsg {
+      color: red;
+      font-size: 12px;
+    }
+    input {
+      width: 100%;
+      height: 62px;
+      border: 1px solid #ddd;
+      border-radius: 6px;
+    }
+    .input {
+      margin: 10px 0;
+    }
+    .input:focus {
+      outline: none;
+    }
+    .inputError {
+      border: 1px solid red;
+      margin: 10px 0;
+    }
+    .inputError:focus {
+      outline: none;
+    }
```

```suggestion
    .inputError {
      border: 1px solid red;
      margin: 10px 0;
      
      &:focus {
         outline: none;
      }
    }
```
어떤 이벤트에 의해 스타일이 변경될때 & 연산자를 사용하는 방식으로 적용해주세요!